addons/web_editor/models/ir_attachment.py

```diff
@@ -47,4 +47,4 @@ def _compute_image_size(self):
     def _get_media_info(self):
         """Return a dict with the values that we need on the media dialog."""
         self.ensure_one()
-        return self.read(['id', 'name', 'mimetype', 'checksum', 'url', 'type', 'res_id', 'res_model', 'public', 'access_token', 'image_src', 'image_width', 'image_height'])[0]
+        return self.read(['id', 'name', 'description', 'mimetype', 'checksum', 'url', 'type', 'res_id', 'res_model', 'public', 'access_token', 'image_src', 'image_width', 'image_height'])[0]
```

Comment:
Also strange this commit, did not we say that no alt is better than a dummy alt ?

Reply:
Description is not a dummy alt, it's empty on most attachments so the attribute won't be set in XML, and it's set to the image's alt_description when downloading images from unsplash

Reply:
:+1: